docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -647,6 +647,12 @@
             <td>Long</td>
             <td>Optional snapshot id used in case of "from-snapshot" or "from-snapshot-full" scan mode</td>
         </tr>
+        <tr>
+            <td><h5>scan.stream.delay.read.duration</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>Duration</td>
+            <td>The delay duration of stream read when scan snapshots</td>
```

Comment:
The delay duration of stream read when scan incremental snapshots.

Reply:
Done